aten/src/ATen/core/interned_strings.cpp

```diff
@@ -140,7 +140,7 @@ bool Symbol::is_attr() const { return ns() == namespaces::attr; }
 bool Symbol::is_aten() const { return ns() == namespaces::aten; }
 bool Symbol::is_cuda() const { return ns() == namespaces::cuda; }
 bool Symbol::is_prim() const { return ns() == namespaces::prim; }
-bool Symbol::is_prims() const { return ns() == namespaces::prims; }
+bool Symbol::is_prims() const { return ns() == namespaces::prims || ns() == namespaces::nvprims; }
```

Comment:
This makes me nervous.

I understand where it's coming from right now. Let's see how upstream folks think about it :sweat: